pkg/render/kube-controllers.go

```diff
@@ -140,8 +185,14 @@ func (c *kubeControllersComponent) Objects() ([]client.Object, []client.Object)
 			secret.CopyToNamespace(common.CalicoNamespace, c.elasticsearchSecret)...)...)
 	}
 
-	if !c.isManagedCluster() && c.elasticsearchUserSecret != nil {
-		objectsToCreate = append(objectsToCreate, secret.ToRuntimeObjects(c.elasticsearchUserSecret)...)
+	if !c.isManagedCluster() && c.alternateUserSecret != nil {
```

Comment:
Are we removing the "hashed" and "alternate" naming scheme? Didn't we replace that?

Reply:
That's right, I forgot to update this after we decided on the naming scheme change, will update now.